rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -476,8 +476,8 @@ void testGetLastPageSortedByDueWithHiddenTasksRemovedFromResult() {
     assertThat(response.getBody()).isNotNull();
     assertThat((response.getBody()).getContent()).hasSize(5);
     assertThat(response.getBody().getRequiredLink(IanaLinkRelations.LAST).getHref())
-        .contains("page=5");
-    assertThat("TKI:000000000000000000000000000000000023")
+        .contains("page=10");
+    assertThat("TKI:000000000000000000000000000000000005")
```

Comment:
whats the reason for the changes in this file? I don't see a matching change

Reply:
It's due to the changed groupSearchBase. Now he uses the org groups as well, resulting in more hits.